examples/headless-demo/index.html

```diff
@@ -97,21 +153,18 @@
 
   <script>
     function generate() {
-      // Parse the XML into a tree.
-      var xmlText = document.getElementById('xml_input').value;
+      var json = document.getElementById('json_input').value;
+      // Create a headless workspace.
       try {
-        var xml = Blockly.Xml.textToDom(xmlText)
-      } catch (e) {
-        alert(e);
-        return;
+        var workspace = new Blockly.Workspace();
+        Blockly.serialization.workspaces.load(JSON.parse(json), workspace);
```

Comment:
I'm struggling to get this working. Workspace seems to have a value if you log it out, but `Blockly.serialization.workspace.load()` doesn't seem to recognise it.

Reply:
Hmm that's weird! Could you post the exact text of the error you're getting?

Reply:
```
TypeError: Cannot read properties of undefined (reading 'workspaces')
    at generate (index.html:160:31)
    at HTMLButtonElement.onclick (index.html:151:34)
```

Reply:
Ah ok! You need to update the [dependency on Blockly](https://github.com/google/blockly-samples/blob/master/examples/headless-demo/package.json) so that it's using v9, which includes the `serialization` namespace =)

You probably want it to look like: `blockly: '^9.0.0'`